shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -452,10 +476,36 @@ bool FlutterWindowsEngine::DispatchSemanticsAction(
 }
 
 void FlutterWindowsEngine::UpdateSemanticsEnabled(bool enabled) {
+#if defined(WINUWP)
+  using AccessibilityBridgeDelegateWindows = AccessibilityBridgeDelegateWinUWP;
+#else
+  using AccessibilityBridgeDelegateWindows = AccessibilityBridgeDelegateWin32;
+#endif  // defined(WINUWP)
+
   if (engine_ && semantics_enabled_ != enabled) {
     semantics_enabled_ = enabled;
     embedder_api_.UpdateSemanticsEnabled(engine_, enabled);
+
+    if (!semantics_enabled_ && accessibility_bridge_) {
```

Comment:
Will it be too much work if we guard this entire logic behind if defined(WINUWP)? if we do that we may be able to remove all the UWP stubs.

Reply:
Ah you mean hide this entire chunk behind an `#ifndef WINUWP`? Generally, we try to keep the platform `#ifdef`s to `#include`s and `using` statements, then keep the implementations in separate files. That way we avoid unexpected syntax errors/variable declaration conflicts, etc.

There are a few places we've been more lax about this (IIRC our filesystem operations are an example of this). In a world where we were implementing UWP immediately, we'd want to do it the way I've done it, so I'd prefer to avoid pre-processor-conditional blocks of code.